Sources/SwiftCompilerPluginMessageHandling/CompilerPluginMessageHandler.swift

```diff
@@ -12,9 +12,23 @@
 
 import SwiftSyntaxMacros
 
+/// Optional features.
+public enum PluginFeature: String {
+  case loadPluginLibrary
+}
```

Comment:
I'd prefer `PluginFeature` to have specific values set so that a global rename doesn't accidentally change the protocol.